src/xccl/ProcessGroupXCCL.cpp

```diff
@@ -803,9 +806,11 @@ c10::intrusive_ptr<Work> ProcessGroupXCCL::collective(
       c10::ListType::create(c10::TensorType::get()), devices);
   work->future_->markCompleted(at::IValue(*work->outputs_));
   auto id = work->trace_id_;
+  auto reset_epoch = work->trace_reset_epoch_;
   work->future_->addCallback(
-      [id](at::ivalue::Future&) {
-        FlightRecorderXCCL::get()->retire_id(id, /*compute_duration*/ false);
+      [id, reset_epoch](at::ivalue::Future&) {
+        FlightRecorderXCCL::get()->retire_id(
+            id, reset_epoch, /*compute_duration*/ false);
```

Comment:
Let’s not pass it for now to maintain compatibility with both before and after the 166970 merge.